code/game/machinery/iv_drip.dm

```diff
@@ -4,10 +4,12 @@
 	anchored = FALSE
 	density = FALSE
 	drag_delay = 1
+	base_pixel_x = 5
```

Comment:
```suggestion
	base_pixel_x = 15
	base_pixel_y = -2
```
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/19eace9b-cdf3-4de3-8d17-c1f0753a85f8)

Reply:
Unfortunately its still kind funky that it'd be so stiff at that point to be straight and then we have a tube that goes too the target.